colorschemes/dark.colorscheme.bash

```diff
@@ -0,0 +1,8 @@
+CLOCK_THEME_PROMPT_COLOR=92
```

Comment:
I know it would be a lot of work, but do you think it makes sense to explain these values, e.g. _92_ is _red_ (I made that up...) A comment in the documentation with a link to the color table would be great. Is there a web site that shows the available values so that it's easier to pick values?